serverless/functions/serverless-functions-getting-started.adoc

```diff
@@ -9,8 +9,7 @@ toc::[]
 :FeatureName: {FunctionsProductName}
 include::modules/technology-preview.adoc[leveloffset=+2]
 
-This guide explains how you can create and manage a function on an {ServerlessProductName} installation by using the `kn` CLI.
-// TODO: add info about developer console at 4.8 when this is supported.
+This guide explains how you can create and manage a function on an {ServerlessProductName} installation by using the `kn` CLI or {product-title} web console.
```

Comment:
in web console we only visualise serverless functions in topology and the creation of serverless functions is not available with web console and only `kn` CLI

Reply:
Yeah, I think I'll reword this sentence since I was kinda thinking the same thing :) thanks Jai!